HandEyeCalibration/Transform/Transform.cpp

```diff
@@ -0,0 +1,343 @@
+/*
+Example to show transformations to/from Transformation Matrix
+
+Zivid primarily operate with a (4x4) Transformation Matrix (Rotation Matrix + Translation Vector). 
+This example shows how to use Eigen to convert to and from:
+  AxisAngle, Rotation Vector, Roll-Pitch-Yaw, Quaternion
+
+ It provides convenience functions that can be reused in applicable applications.
+*/
+
+#include <Eigen/Core>
+#include <Eigen/Dense>
+#include <Eigen/Geometry>
+
+#include <opencv2/core/core.hpp>
+
+#include <iostream>
+
+enum class RotationConvention
+{
+    XYZ_Intrinsic,
+    XYZ_Extrinsic,
+    ZYX_Intrinsic,
+    ZYX_Extrinsic,
+    NOF_ROT
+};
+constexpr size_t nofRotConv = static_cast<size_t>(RotationConvention::NOF_ROT);
+
+struct RollPitchYaw
+{
+    RotationConvention convention;
+    std::string shortDesc;
```

Comment:
`shortDesc` here seems to be a string representation of `RotationConvention`. That string representation is not really a property of `RollPitchYaw`, but of `RotationConvention`. Also it should not be possible to have different `shortDesc` for the same `RotationConvention` like it is now. Maybe create a function for this instead, then it can also be used independently of `RollPitchYaw`.

Something like
```c++

std::string toString(RotationConvention convention)
{
    switch(convention)
    {
        case RotationConvention::XYZ_Intrinsic: return "XYZ_Intrinsic";
        case RotationConvention::XYZ_Extrinsic: return "XYZ_Extrinsic";
        case RotationConvention::ZYX_Intrinsic: return "ZYX_Intrinsic";
        case RotationConvention::ZYX_Extrinsic: return "ZYX_Extrinsic";
        default: throw std::invalid_argument("Invalid RotationConvention");
    }
}
```

Then you can reduce `RollPitchYaw` to just `convention` and `rollPitchYaw`, and it becomes easier to both understand and initialize.